pom.xml

```diff
@@ -129,6 +130,11 @@
 				<artifactId>guava</artifactId>
 				<version>23.2-jre</version>
 			</dependency>
+			<dependency>
+				<groupId>org.projectlombok</groupId>
+				<artifactId>lombok</artifactId>
+				<version>${lombok.version}</version>
```

Comment:
The managed version should be scope provided

Reply:
Done